src/components/Layouts/Primary/TopBar/TopBar.tsx

```diff
@@ -58,12 +62,16 @@ const TopBar = ({
             />
           </NextLink>
           <Hidden mdDown>
-            <Box ml={10} flexGrow={1}>
-              <NavMenu />
-            </Box>
-            <SearchMenu />
-            <AddMenu />
-            <NotificationMenu />
+            {accountListId && (
```

Comment:
Thanks for fixing this! I'd noticed that trying to create a task if you don't have an account list selected would cause errors, but I never got around to fixing it.